api/api.go

```diff
@@ -354,14 +354,56 @@ func (api *API) deleteEntityHandler(c echo.Context) error {
 
 // Getting all labels for a single entity.
 func (api *API) entityLabelsHandler(c echo.Context) error {
-	// @todo: implement this
-	return nil
+	if err := validateParams(c, true); err != nil {
+		return handleError(err)
+	}
+	entityType := c.Param("type")
+	entityId := c.Param("id")
+
+	q, _ := query.Translate("_id=" + entityId)
```

Comment:
Should you be using queryForId here? Line 572 (comment for queryForId) mentions that "_id is not a valid field name to pass to query.Translate", and queryForId seems to be used in the other handlers.

Reply:
Most likely. It seemed to work when I tested it manually. Will take a closer look. Thanks for catching this.

Reply:
Changed. I think queryForId might not be necessary any longer. The real Kubernetes label selector pkg doesn't allow _id, but I wrote a new one that does. I think that's why the literal "_id=" was working. This way works, too. Maybe we'll remove it later if we confirm that our custom KLS parser is handling this.